src/components/transfer/Transfer.vue

```diff
@@ -5,12 +5,25 @@
         <br />
         <Loader v-model="isLoading" :status="status" />
         <div class="box">
-          <p class="title is-size-3">
+          <div class="info">
+            <p class="title is-size-3">
             Transfer {{ unit }}
-          </p>
+            </p>
+            <span class="info--currentPrice" title='Current price'>${{this.$store.state.fiatPrice[this.coinId]['usd']}} </span>
+          </div>
```

Comment:
You should write a [Vuex getter](https://vuex.vuejs.org/guide/getters.html) for this